authentik/root/settings.py

```diff
@@ -188,11 +188,16 @@ def j_print(event: str, log_level: str = "info", **kwargs):
     "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
 }
 
+REDIS_PROTOCOL_PREFIX = "rediss://"
+if CONFIG.y_bool("redis.tls", False):
+    REDIS_PROTOCOL_PREFIX = "redis://"
```

Comment:
I think the values for this are swapped around, the `y_bool` call returns a bool and the second argument is only a default, not a comparison